Modules/LancetHardwareDevice/AriemediDll/include/ToolRegi.h

```diff
@@ -0,0 +1,123 @@
+#ifndef TOOLREGI_HPP
+#define TOOLREGI_HPP
+
+#ifdef ARMDCOMBINEDAPI_EXPORTS
+#define ARMDCOMBINED_API __declspec(dllexport)
+#else
+#define ARMDCOMBINED_API __declspec(dllimport)
+#endif
+
+#include <string>
+#include <vector>
+#include <iostream>
+#include <sstream>
+#include <fstream>
+
+namespace TransformationStatus
+{
+	enum values
+	{
+		Enabled = 0x0000,
+		PartiallyOutOfVolume = 0x0001,
+		OutOfVolume = 0x0002,
+		PartiallyMatched = 0x0003,
+		TooFewMarkers = 0x0004,
+		BadTransformFit = 0x0005,
+		// = 0x0006,
+		// = 0x0007,
+		// = 0x0008,
+		ToolMissing = 0x0009,
+	};
+
+	//converts the error code to std::string representation
+	ARMDCOMBINED_API std::string toString(uint16_t errorCode);
+}
+
+struct ToolPlaneMarker
+{
+	std::vector<std::vector<double>> regiSet;
+	std::vector<std::vector<double>> regiDifSet;
+};
+
+struct MarkerPosition
+{
+	double P[4];//marker position (x,y,z,1)
+	int leftExposure = 0;//marker exposure
+	int rightExposure = 0;//marker exposure
+};
+
+struct Transformation
+{
+	uint16_t status = TransformationStatus::ToolMissing;//transformation status
```

Comment:
Need a Code indentation alignment.

Reply:
this is official head file ,i cannot change it